src/main/java/io/xstefank/wildfly/bot/util/PullRequestLogger.java

```diff
@@ -0,0 +1,116 @@
+package io.xstefank.wildfly.bot.util;
+
+import org.jboss.logging.Logger;
+import org.kohsuke.github.GHPullRequest;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class PullRequestLogger {
+    private static final Map<String, Logger> delegates = new HashMap<>();
+    private final String Fqcn;
```

Comment:
fields start with lower case, also you can use full name

Reply:
I believe it stands for fully qualified canonical name, I have seen it used as field as `FQCN` or `Fqcn`. So I wanted to stay true to original's logger variable name.